prow/pod-utils/clone/clone.go

```diff
@@ -142,11 +146,32 @@ func (g *gitCtx) gitCommand(args ...string) cloneCommand {
 	return cloneCommand{dir: g.cloneDir, env: g.env, command: "git", args: args}
 }
 
+var (
+	fetchRetries = []time.Duration{
+		100 * time.Millisecond,
+		200 * time.Millisecond,
+		400 * time.Millisecond,
+		800 * time.Millisecond,
+		2 * time.Millisecond,
```

Comment:
Is this supposed to be 2 milliseconds, or 2 seconds?

Reply:
Done